partner_helper/models/partner.py

```diff
@@ -2,11 +2,12 @@
 # Author: Sébastien BEAU <sebastien.beau@akretion.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import models
+from odoo import _, models
+from odoo.exceptions import UserError
 
 
-def split_char(char, output_number, size):
-    words = char.split(" ")
+def split_char(char, output_number, size, strict):
```

Comment:
Isn't it better to set strict=False in signature in case some modules use this method directly ?

Reply:
yes, it avoid to break things

Reply:
done